docs/packaging/creating-a-new-package.md

```diff
@@ -104,4 +170,40 @@ Once the build completes, your directory should now include the following files:
 └── tree-2.1.1-1-1-x86_64.eopkg
 ```
 
-All these files _except_ the `*eopkg` file should be included when you submit the package for review.
+All these files _except_ the `*eopkg` file(s) should be included in your pull request. You will remove them after testing the package.
+
+Once your package has built successfully, you will need to [test it](testing-a-package).
+
+## Commit your Changes
+
+Check the [changes in your files](git-basics).
+
+[Add / remove files as necessary to the commit](git-basics.md). Then, **check your branch**.
+
+Run `git status`. Make sure all the files you changed are staged, and that there are no untracked files. When all it well, run `git commit`.
```

Comment:
The `##` are stripped by default from the commit message :sweat: 

```suggestion
Run `git status`. Make sure all the files you changed are staged, and that there are no untracked files. When all is well, run `git commit --cleanup=scissors`.
```

Reply:
After discussion in Matrix, I'll change the recommended template to use **bolding** instead of the ##. This avoids having to use git flags, and keeps it simpler.